R/derive_params_growth_age.R

```diff
@@ -262,25 +266,18 @@ derive_params_growth_age <- function(dataset,
       SD2pos = (M * (1 + 2 * L * S)^(1 / L)),
       SD3pos = (M * (1 + 3 * L * S)^(1 / L)),
       SD2neg = (M * (1 - 2 * L * S)^(1 / L)),
-      SD3neg = (M * (1 - 3 * L * S)^(1 / L))
+      SD3neg = (M * (1 - 3 * L * S)^(1 / L)),
+      age_bins = map(metadata_age, ~ set_bins(.x, breaks = bins$breaks, labels = bins$labels))
```

Comment:
`age_bins` should be defined as a global variable, I get the following message while running the checks "_derive_params_growth_age: no visible binding for global variable 'age_bins'_ "